gulpfile.js

```diff
@@ -2,32 +2,11 @@ var gulp = require('gulp');
 var gulpif = require('gulp-if');
 var del = require('del');
 var dom = require('gulp-dom');
-var terser = require('terser');
```

Comment:
We should probably remove these unused dependencies from the package.json and lock file.

Reply:
good catch